src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,214 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+
+    private Node<T> head;
+    private Node<T> tail;
+    private int size;
+
+    private static class Node<T> {
```

Comment:
Make `Node<T>` an inner class. Currently, it is a static nested class. Inner classes should be non-static and placed at the end of the outer class.